routes/create.js

```diff
@@ -1,18 +1,51 @@
 const express = require("express");
+const Story = require("../models/Story");
+const User = require("../models/User");
+const Chapter = require("../models/Chapter");
 const router = express.Router();
 
 // @desc Create story page
 // @route POST /create
 // @access Private
 router.post("/", (req, res) => {
-  res.send("it should take book details"); // this should return back a storyID which will be used to create the chapter 
+  router.post("/", async(req, res) => {
+    try {
+              await Story.create(req.body, async function (err, story) {
```

Comment:
Story Model has Username and UserId field too which is required, make sure to extract them from user data which will be available once u apply passport auth on the route itself.